content/api/commands/intercept.md

```diff
@@ -2,165 +2,100 @@
+#### **<Icon name="angle-right"></Icon> method** **_(String)_**
 
-#### **<Icon name="angle-right"></Icon> method** **_(`string`)_**
+Specify the [HTTP request method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) (`GET`, `POST`, `PUT`, `PATCH`, `DELETE`, etc.) to match on.
 
-Specify the HTTP method to match on.
+<Alert type="bolt">
 
-```ts
-cy.intercept('POST', 'http://example.com/widgets', {
-  statusCode: 200,
-  body: 'it worked!',
-})
-```
+If no `method` is provided, Cypress will match _all_ HTTP methods (`*` wildcard) by default.
+
+</Alert>
 
 #### **<Icon name="angle-right"></Icon> routeMatcher** **_(`RouteMatcher`)_**
 
-`routeMatcher` is an object used to match which incoming HTTP requests will be handled by this route.
+For more refined matching, you can pass in the `routeMatcher` to specify matching HTTP requests beyond `url` and `method` - like headers and query parameters. See the chart below.
 
-All properties are optional. All properties that are set must match for the route to handle a request. If a `string` is passed to any property, it will be glob-matched against the request using [`minimatch`](https://github.com/isaacs/minimatch).The available `routeMatcher` properties are listed below:
+All properties are optional and except when specified, can be a string for an exact match or a pattern (glob/minimatch or regular expression):
 
-```ts
-{
-  /**
-   * Match against the username and password used in HTTP Basic authentication.
-   */
-  auth?: { username: string | RegExp, password: string | RegExp }
-  /**
-   * Match against HTTP headers on the request.
-   */
-  headers?: {
-    [name: string]: string | RegExp
-  }
-  /**
-   * Match against the requested HTTP hostname.
-   */
-  hostname?: string | RegExp
-  /**
-   * If 'true', only HTTPS requests will be matched.
-   * If 'false', only HTTP requests will be matched.
-   */
-  https?: boolean
-  /**
-   * Match against the request's HTTP method.
-   * @default '*'
-   */
-  method?: string | RegExp
-  /**
-   * If `true`, this will pass the request on to the next `RouteMatcher` after the request handler completes.
-   * Can only be used with a dynamic request handler.
-   * @default false
-   */
-  middleware?: boolean
-  /**
-   * Match on request path after the hostname, including query params.
-   */
-  path?: string | RegExp
-  /**
-   * Matches like 'path', but without query params.
-   */
-  pathname?: string | RegExp
-  /**
-   * Match based on requested port, or pass an array of ports
-   * to match against any in that array.
-   */
-  port?: number | number[]
-  /**
-   * Match on parsed querystring parameters.
-   */
-  query?: {
-    [key: string]: string | RegExp
-  }
-  /**
-   * Match against the full request URL.
-   */
-  url?: string | RegExp
-}
-```
+<!-- TODO add an example link for each option -->
 
-`routeMatcher` usage examples:
+| Option     | Description                                                                             |
```

Comment:
I feel like having a `type` column would make this a lot easier to read. Like https://docs.cypress.io/api/plugins/browser-launch-api#Syntax

Reply:
It would be nice to standardize on these columns (and names) for object properties across all of our docs